dimagi/utils/post.py

```diff
@@ -54,7 +54,14 @@ def simple_post(data, url, content_type="text/xml", timeout=60, headers=None):
     })
     if headers:
         default_headers.update(headers)
-    return requests.post(url, data, headers=default_headers, timeout=timeout)
+    kwargs = {
+        "headers": default_headers,
+        "timeout": timeout,
+    }
+    if auth:
+        kwargs["auth"] = auth
```

Comment:
I wasn't sure if `requests.post(url, data, auth=None)` did something different than `requests.post(url, data)`, so I'm playing it safe and not passing the kwarg if it is `None`. (e.g., would `auth=None` override auth headers in the headers kwarg?)